app/views/contexts/thank_you_context.py

```diff
@@ -36,9 +39,25 @@ def build_thank_you_context(
         "submission_text": submission_text,
         "metadata": metadata,
         "guidance": guidance_content,
+        "view_submitted_response": build_view_submitted_response(schema, submitted_at),
     }
 
 
+def build_view_submitted_response(schema, submitted_at):
```

Comment:
The method name should end with `_context` to be consistent with the other method names. 

Arguably they shouldn't start with `build_`, but that is a wider naming issue that should be addressed in a different pull request.